tests/app_id/src/app.ts

```diff
@@ -59,64 +61,45 @@ async function main () {
   // Create the API and wait until ready
   const api = await createApi(); 
 
+  const keyring = new Keyring({ type: 'sr25519'});
+  const mnemonic = `inject fiscal misery fiscal success weasel black tube satisfy rural sauce reveal`;
```

Comment:
Let's avoid pushing mnemonic on repo.

Reply:
Fixed.

Reply:
@prabal-banerjee @MiguelDD1 would you like to add the new script here for getting back the data without any extrisic ID? or leave it for now ?

Reply:
@kroos47 Let's add it in a new small PR.